BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -404,16 +413,39 @@ public override void OnStart(StartState state)
                     pEemitter.Current.emit = false;
                 }
 
-            if (HighLogic.LoadedSceneIsFlight)
+            reloadableRail = part.FindModuleImplementing<ModuleMissileRearm>();
```

Comment:
If you use the optimisation for `ModuleMissileRearm` above, then this should hook into it too to avoid unnecessary duplication.